docs/core/whats-new/dotnet-10/libraries.md

```diff
@@ -328,3 +328,363 @@ A community contribution improved the performance of <xref:System.IO.Compression
 
 - Eliminates repeated allocation of ~64-80 bytes of memory per concatenated stream, with additional unmanaged memory savings.
 - Reduces execution time by approximately 400 ns per concatenated stream.
+
+## Preview 7 additions
+
+The following features were added in Preview 7:
+
+- [Launch Windows processes in new process group](#launch-windows-processes-in-new-process-group)
+- [AES KeyWrap with Padding (IETF RFC 5649)](#aes-keywrap-with-padding-ietf-rfc-5649)
+- [Post-quantum cryptography updates](#post-quantum-cryptography-updates)
+- [PipeReader support for JSON serializer](#pipereader-support-for-json-serializer)
+- [WebSocketStream](#websocketstream)
+- [TLS 1.3 for macOS (client)](#tls-13-for-macos-client)
```

Comment:
Remove this. You'll want to integrate the Preview 7 features into the main article.